### PR TITLE
stats: adding basic stats overview with fancy numbers

### PR DESCRIPTION
**Pull request by anthraxx**

### CODE REVIEW

**Comment by anthraxx** [comment]
still WIP, as the test cases are missing. but you can comment if you have any feedback or missing numbers

**Comment by jelly** [comment]
The only thing I can think of is maybe a graph / stat of how long it takes till an AVG is created and published, would be good to see how long our process takes

File: Makefile [hunk 1]
```diff
@@ -18,9 +18,12 @@ submodule:
 update: setup
 	./update
 
-test: setup
+test coverage: setup
 	PYTHONPATH=".:${PYTHONPATH}" ${PYTEST} ${PYTEST_INPUT} ${PYTEST_OPTIONS} ${PYTEST_COVERAGE_OPTIONS}
 
+open-coverage: coverage
+	${BROWSER} test/coverage/index.html
+
```
Comment (jelly) [inline Makefile]: neat idea :)

**Comment by anthraxx** [comment]
i call this done for now, please do your final reviews to merge

**Pull request merged**